### ISSUE TITLE
[Issue]: File integrity error when installing version 0.41.0 using CurseForge client

### ISSUE THREAD

**Issue by Zarkov2** [The-Nexus-Project/Limitless-7#90]
### What happened?

Installation fails with a "Mod file integrity check failed" error message and the entire instance folder is immediately deleted.

### Pack Version

Version: 0.41.0

### To Reproduce

1. Attempt to create new instance using version 0.41.0 (not an update).
2. After almost all mods are downloaded, error message appears.

### Are you using OptiFine or Oculus?

- [ ] Using OptiFine
- [ ] Using Oculus
- [X] No. None of them. Unmodified.

### Are you on a Server?

- [ ] Yes.
- [X] No.

### Relevant log output / crash report.

```shell
Client agent log excerpt:

{
  "timestamp": "2024-03-05T21:36:46.7620127+01:00",
  "level": "Info",
  "thread": 25,
  "memory": 0.0,
  "message": "Downloading: https://edge.forgecdn.net/files/5146/429/apotheotic_additions1.5.jar to C:\\Users\\Xxxxxxxxx\\AppData\\Local\\Temp\\CF\\Cm58nqa2GESQouMsLkd9A\\7667.zip"
},
{
  "timestamp": "2024-03-05T21:36:46.9186968+01:00",
  "level": "Error",
  "thread": 25,
  "memory": 0.0,
  "message": "Failed to verify file C:\\Users\\Xxxxxxxxx\\AppData\\Local\\Temp\\CF\\Cm58nqa2GESQouMsLkd9A\\7667.zip integrity using SHA1. Error: Operation did not complete successfully because the file contains a virus or potentially unwanted software. : 'C:\\Users\\Xxxxxxxxx\\AppData\\Local\\Temp\\CF\\Cm58nqa2GESQouMsLkd9A\\7667.zip'"
},
```

### Anything else to add?

The file "apotheotic_additions1.5.jar" seems to be the problem. LL7 version 0.40.0 installed without problems.

**Comment by Gbergz** [comment]
This is a false positive from your antivirus, that program probably had an update and is now flagging one of the jar as a potential threat (while it's not), changing launcher from curseforge might help until fixed.

**Comment by Zarkov2** [comment]
Thank you for the _extremely_ prompt response and advice! I was in do not disturb mode so I didn't see the notifications. I can confirm Windows Defender considered "7667.zip" a problem.

**Comment by Gbergz** [comment]
No worries!
Yeah I'm in contact with the mod dev and its indeed a false positive as everything is clean but as a jar file there's issues coming up, they're investigating. 

Thanks for the report though, leaving this open for visibility.

**Comment by Gbergz** [comment]
Fixed in v0.41.1
https://legacy.curseforge.com/minecraft/modpacks/tnp-limitless-7/files/5166971
